src/posthog-core.js

```diff
@@ -873,10 +874,12 @@ PostHogLib.prototype.identify = function (new_distinct_id, userPropertiesToSet,
     ) {
         this.capture(
             '$identify',
-            {
-                distinct_id: new_distinct_id,
-                $anon_distinct_id: previous_distinct_id,
-            },
+            this.get_config('send_anon_distinct_id')
+                ? {
+                      distinct_id: new_distinct_id,
+                      $anon_distinct_id: previous_distinct_id,
```

Comment:
Quick flyby that if we're not sending `$anon_distinct_id`, we don't need to send it for feature flags either (L888)